File: Section1 - Basics/Patterns/pattern19.js
function printTriangle(n) {
	let str = "";
	for (let i = n; i > 0; i--) {
		for (let k = 0; k < i; k++) {
			str += "*";
		}
		for (let j = i; j < n; j++) {
			str += "  ";
		}
		for (let l = 0; l < i; l++) {
			str += "*";
		}
		str += "\n";
	}

	for (let i = 0; i < n; i++) {
		for (let k = 0; k <= i; k++) {
			str += "*";
		}
		for (let j = n; j > i + 1; j--) {
			str += "  ";
		}
		for (let l = 0; l <= i; l++) {
			str += "*";
		}
		str += "\n";
	}
	console.log(str);
}

printTriangle(5);

File: Section1 - Basics/Patterns/pattern19.java
void printTriangle(int n) {
    // code here
    for(int i=n; i>0; i--) {
        for(int j=0; j<i; j++) {
            System.out.print("*");
        }
        for(int j=i; j<n; j++) {
            System.out.print("  ");
        }
        for(int j=0; j<i; j++) {
            System.out.print("*");
        }
        System.out.print("\n");
    }

    for(int i=0; i<n; i++) {
        for(int j=0; j<=i; j++) {
            System.out.print("*");
        }
        for(int j=n; j>i+1; j--) {
            System.out.print("  ");
        }
        for(int k=0; k<=i; k++) {
            System.out.print("*");
        }    
        System.out.print("\n");
    }    
}